src/stability_sdk/animation.py

```diff
@@ -62,8 +52,8 @@ def display_frame(image: np.ndarray):
 # TO DO: "prompt" argument has a bit of a logical collision with animation prompts
 class BasicSettings(param.Parameterized):
     #prompt  = param.String(default="A beautiful painting of yosemite national park, by Neil Gaiman", doc="A string")
-    height  = param.Integer(default=512, bounds=(256, 1024), doc="Output image dimensions. Will be resized to a multiple of 64.")
-    width   = param.Integer(default=512, bounds=(256, 1024), doc="Output image dimensions. Will be resized to a multiple of 64.")
+    height  = param.Integer(default=512, doc="Output image dimensions. Will be resized to a multiple of 64.")
```

Comment:
the "bounds" annotation here is what cues this parameter to be a slider. we could use `softbounds` as a way of generating the slider without restricting the values the parameter can take. I think we probably will want to set strict upper and lower bounds on this though. image generation definitely breaks on images that are too large, pretty sure there are issues if they're too small too but honestly not sure. I'm not sure if we have anything documented in the backlog for stitching or otherwise supporting arbitrary resolution images: we document and triage that if haven't already.

Reply:
oh the problem was that with slider it becomes incredibly difficult to put in the resolution you want. you know the numbers so typing them in is much easier than messing with slider and trying to get it to the value. slider isn't a good UI for width and height. drop down a little better but direct entry i think is best here.

Reply:
hmm... interesting dilemma. we'll optimize for notebook user experience for the moment, but if we end up building platform documentation from param, the bounds thing will be very useful. there's probably a way to override the default `panel` components, but we can deal with that later